storybookjs/storybook
# UI: Single story hoisting in sidebar

Issue: #8255

## What I did

This collapses single-story components in the tree view. Such components must have **exactly one child story** and this one story must have **exactly the same name** as the component itself. If these criteria are met, the component node is removed from the tree, and the story is put in its place.

## How to test

- Is this testable with Jest or Chromatic screenshots? Yes
- Does this need a new example in the kitchen sink apps? Probably
- Does this need an update to the documentation? Yes

If your answer is yes to any of these, please make sure to include it in your PR.
